aws/resource_aws_lb_listener.go

```diff
@@ -236,6 +291,62 @@ func resourceAwsLbListenerUpdate(d *schema.ResourceData, meta interface{}) error
 		return errwrap.Wrapf("Error modifying LB Listener: {{err}}", err)
 	}
 
+	var carnList []*string
+	if carns, ok := d.GetOk("additional_certificate_arns"); ok {
+		carnList = expandStringList(carns.(*schema.Set).List())
+	}
+
+	awsCerts, err := elbconn.DescribeListenerCertificates(&elbv2.DescribeListenerCertificatesInput{
+		ListenerArn: aws.String(d.Id()),
+	})
+	if err != nil {
+		return errwrap.Wrapf("Error retrieving ListenerCertficates: {{err}}", err)
```

Comment:
`errwrap` is typically useful in situations where we need to keep track of the error type. In this case error will only ever be escalated to the UI where all that matters is strings ("stringifyed" error message), not the original type.

In other words `fmt.Errorf()` will do the job, IMO.